src/main/java/generics/Box.java

```diff
@@ -0,0 +1,75 @@
+package generics;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+public class Box {
+    private final LinkedList list = new LinkedList();
+    private static final float APPLE = 1.0F;
+    private static final float ORANGE = 1.5F;
+
+    public LinkedList getList() {
```

Comment:
```suggestion
    public List getItems() {
```
getItems, getAll, getElements 

getList - не дуже гарно, завтра замість List будеш повертати Set/Collection і нічого не буде працювати вже.